PUBLISHING.md

```diff
@@ -4,7 +4,7 @@ This page describes how to publish a new version of plugin and all related artif
 
 Steps:
 
-1. Create a PR with version increment in [gradle.properties](../gradle.properties) and in [documentation](../README.adoc). Please follow [semver guidelines](https://semver.org/):
+1. Create a PR with version increment in [gradle.properties](../gradle.properties) and in [documentation](../README.md). Please follow [semver guidelines](https://semver.org/):
```

Comment:
Links to files need fixing

Reply:
It has already been fixed; I renamed the file `README.adoc` to `README.md` and this has been shown as a change 👍

Reply:
Yes , but you also moved the file to the parent directory.  You need to remove ../